Geometry/Heron.java

```diff
@@ -0,0 +1,37 @@
+//github.com/binoy-d
+
+import java.util.Scanner;
+
+public class Heron{
+
+	public static void main(String[] args) {
+
+		System.out.println("Welcome to the Heron's Formula Calculator!");
+
+		Scanner scan = new Scanner(System.in);
+
+    //Prompt user for side lengths and store values
+		System.out.print("What is the length of side 1?");
+		double a = scan.nextDouble();
+
+		System.out.print("What is the length of side 2? ");
+		double b = scan.nextDouble();
+
+		System.out.print("What is the length of side 3?");
+		double c = scan.nextDouble();
+
+		System.out.println("Calculating...");
+
+    //Calculate the area
+		double s = (double)(0.5*(a+b+c));
+		double answer = (double)Math.sqrt(s*(s-a)*(s-b)*(s-c));
+
+    //Print out zeroes calculated from the Heron's formula
```

Comment:
Comment should say "Print out area calculated from the Heron's formula" instead of "zeroes."

Reply:
takrn care of